plotter/main.go

```diff
@@ -49,22 +51,61 @@ var examples = []struct {
 }
 
 var formats = []string{
-	".eps",
-	".pdf",
-	".svg",
-	".png",
-	".tiff",
-	".jpg",
+	"eps",
+	"pdf",
+	"svg",
+	"png",
+	"tiff",
+	"jpg",
+}
+
+type canvasWriter interface {
```

Comment:
are we talking about the same thing? (I am talking about `canvasWriter` which happens to be exactly like `vg/draw.CanvasWriterTo`)